server/node.go

```diff
@@ -426,10 +426,16 @@ func (n *Node) gossipCapacities() {
 // ctx.ScanInterval and store the status in the db.
 func (n *Node) startStoresScanner(stopper *util.Stopper) {
 	stopper.RunWorker(func() {
+		// Pick the smalled of the two intervals.
+		var minScanInterval time.Duration
+		if n.ctx.ScanInterval <= n.ctx.ScanMaxIdleTime {
```

Comment:
ScanMaxIdleTime is allowed to be zero, in which case it should be ignored. If it is non-zero, it wouldn't make sense to set it to a value greater than ScanInterval. So if we want to derive a store scan interval from the range scanner configuration, we should use MaxIdleTime if it is non-zero, and ScanInterval otherwise. But I think the two cases are sufficiently different that it doesn't make sense to use the same configuration for both; the store scanner should have its own independent setting.